Cargo.toml

```
@@ -98,7 +98,7 @@ uninlined_format_args = "warn"
 unreadable_literal = "warn"
 
 [dependencies]
-hermit-macro = { path = "hermit-macro" }
+hermit-macro = { version = "0.1.0" }
```

Let's enforce the specific version and allow using the path if possible.

```suggestion
hermit-macro = { version = "=0.1.0", path = "hermit-macro" }
```